Makefile

```diff
@@ -263,7 +263,7 @@ run-clojure: export TARGET := clojure
 run-clojure: ##@run Watch for and build Clojure changes for mobile
 	yarn shadow-cljs watch mobile
 
-run-metro: export TARGET := clojure
+run-metro: export TARGET := android
```

Comment:
What caused this change?

Reply:
**Android** ( _the bane of my existence_ )
When running `make run-clojure` and then `make run-metro` ( with `clojure` target before ) and then `make run-android` 
The app deployed on simulator would point to outdated `node_modules` and the only way to fix that would be to kill metro terminal and run it again. 
I realised that by switching the target from `clojure` to `android` that problem went away.

This may not seem like an ideal solution, but it got me to not have to kill and restart metro terminal when building for android locally.

Reply:
I see, that's kinds of an annoying side effect.